qless-java/src/main/java/com/moz/qless/Queue.java

```diff
@@ -23,6 +22,91 @@
 import redis.clients.jedis.Transaction;
 
 public class Queue {
+
+  public static class JobSpec {
```

Comment:
Does `JobSpec` really belong as a nested class under `Queue`? This seems like a stretch to me.